packages/api-gw/src/index.ts

```diff
@@ -0,0 +1,75 @@
+import Fastify, { FastifyInstance } from 'fastify';
+import { MongoDb } from "../../service-db/src/index";
+
+const server: FastifyInstance = Fastify({})
+
+import { User } from "../../service-users/src/entities/User";
+
+server.get('/users', async (req, res) => {
+  MongoDb.getConnection().then(async function (connect) {
```

Comment:
Diego, just to understand. This code would go in each service, be it, Users, links or campaigns. Here in api-gw, there is only the logic that would bring the data that each user service, link service, or campaign service has previously requested to service db. It's right?

Reply:
Yes. it's a example code